src/main/scala/is/hail/utils/package.scala

```diff
@@ -473,4 +473,21 @@ package object utils extends Logging
     }
     expansion
   }
+
+  def loadFromResource[T](file: String)(reader: (InputStream) => T): T = {
+    val resourceStream = Thread.currentThread().getContextClassLoader.getResourceAsStream(file)
+    assert(resourceStream != null, s"Error while locating file `$file'")
+
+    try {
```

Comment:
The resourceStream must be defined outside of the try/finally block. Otherwise, it's not accessible to the finally code block.